scheduler/admin.py

```diff
@@ -112,6 +112,19 @@ def run_job_now(self, request, queryset):
                 **kwargs
             )
             job_names.append(obj.name)
+        self.message_user(request, "The following jobs have been enqueued: %s" % (', '.join(job_names),))
+
+    @admin.action(description="Run now", permissions=('change',))
+    def run_job_now(self, request, queryset):
+        job_names = []
+        for obj in queryset:
+            kwargs = obj.enqueue_args()
+            obj._get_rqueue().enqueue(
+                obj.callable_func(),
+                *obj.parse_args(),
```

Comment:
This is not how to run a job, at least not anymore.
See schedule method:
```
 def schedule(self) -> bool:
        if not self.ready_for_schedule():
            return False
        schedule_time = self._schedule_time()
        kwargs = self.enqueue_args()
        job = self._get_rqueue().enqueue_at(
            schedule_time,
            tools.run_job,
            args=(self.JOB_TYPE, self.id),
            **kwargs, )
        self.job_id = job.id
        return True
```

Reply:
Thanks! will update.

Reply:
@cunla Question: what do you want changed? my new action/function or also the current one? I assume that you want both.